monkey/infection_monkey/network/info.py

```diff
@@ -119,15 +122,19 @@ def get_routes():  # based on scapy implementation for route parsing
 
 
 def get_free_tcp_port(min_range=1024, max_range=65535):
-    min_range = max(1, min_range)
-    max_range = min(65535, max_range)
 
-    in_use = [conn.laddr[1] for conn in psutil.net_connections()]
+    in_use = sorted([conn.laddr[1] for conn in psutil.net_connections()])
```

Comment:
Why not just

```suggestion
    in_use = set([conn.laddr[1] for conn in psutil.net_connections()])
```

Then you don't need `in_sorted_sequence()` and lookups are O(1)-ish instead of O(log n).

Reply:
you can do one better and use a set comprehension

`in_use = {conn.laddr[1] for conn in psutil.net_connections()}`

Reply:
Awesome!